### ISSUE TITLE
Source maps have broccoli tmp path prefixed. Maps don't work anymore.

### ISSUE THREAD

**Issue by RobIsHere** [aexmachina/ember-cli-sass#160]
Using ember-cli-sass version 6.1.3, my editor.css.map looks like this:
```
{
	"version": 3,
	"file": "editor.css",
	"sourceRoot": "/var/editor/app/styles",
	"sources": [
		"../../sass_compiler-input_base_path-Ku4exJlp.tmp/0/app/styles/app.scss",
		"../../sass_compiler-input_base_path-Ku4exJlp.tmp/2/bourbon/core/_bourbon.scss",
		"../../sass_compiler-input_base_path-Ku4exJlp.tmp/2/bourbon/core/bourbon/helpers/_buttons-list.scss",
		"../../sass_compiler-input_base_path-Ku4exJlp.tmp/2/bourbon/core/bourbon/helpers/_scales.scss",
		"../../sass_compiler-input_base_path-Ku4exJlp.tmp/2/bourbon/core/bourbon/helpers/_text-inputs-list.scss",
		"../../sass_compiler-input_base_path-Ku4exJlp.tmp/2/bourbon/core/bourbon/settings/_settings.scss",
		"../../sass_compiler-input_base_path-Ku4exJlp.tmp/2/bourbon/core/bourbon/validators/_contains.scss",

and so on...
```

The temporary base directory of broccoli somehow isn't given to sass, so it can't figure out the right paths maybe?

Chrome doesn't work with the source maps.

**Comment by aexmachina** [comment]
That's working fine for me. Please be more specific. In what way does Chrome not work?

**Comment by RobIsHere** [comment]
It worked for me after resetting chrome at dev-settings. It broke when i upgraded.

But i still can't add the folder to the chrome workspace (because it doesn't resolve the directory structure?)

**Comment by RobIsHere** [comment]
By the way I'm using ember-component-css. But also "Non component" things like bourbon sass has the broccoli paths

**Comment by aexmachina** [comment]
No idea what that first statement means.

Resolving a network resource to a folder in the workspace is handled through "Map to file system resource" when you click on a file in the sources view. I'd suggest you remove all folders, confirm that you are seeing the source files - that'll confirm that the source maps are working. Then you can work on getting chrome happy with the folders and mappings.

You might find that ember-component-css doesn't play nice with source maps, which could explain your issue.

**Comment by RobIsHere** [comment]
I'll try to do some debugging on my own. Maybe it's specific to my setup

**Comment by drouhard** [comment]
@RobIsHere Any resolution? I can't get source maps working either, and my css.map file also has lines like
`"sources": [
"../../sass_compiler-input_base_path-Ku4exJlp.tmp/0/app/styles/app.scss",
...`

**Comment by RobIsHere** [comment]
I'm using a custom toolchain with an on-save hook of my IDE now. Which works way better because I can just drop in whatever tools work best.

I spent some time on investigation back then, but it lead to a lot of cases in the code paths that could just go wrong and also the sourcemaps specs have "consecutive faults by design". To really solve everything in contrary to fixing it just for the moment, a huge test suite would be the only way. After realizing the huge amount of work needed, I looked for other tools.

It's not an isolated problem of ember-cli-sass. I want to point this out. It's more about the integration of different tools in the buildchain. E.g. post-css (for ie11) on top of ember-cli-sass breaks, but both individually work.
